### PR TITLE
Improve performance

### PR DESCRIPTION
**Pull request by andreme**
## Griddle major version
1.8

## Why these changes are made
To make loading rows faster

## Are there tests?
Yes

### CODE REVIEW

File: src/reducers/dataReducer.js [hunk 1]
```diff
@@ -134,12 +133,10 @@ export function GRIDDLE_TOGGLE_COLUMN(state, action) {
 }
 
 export function GRIDDLE_UPDATE_STATE(state, action) {
-  const transformedData = transformData(action.newState.data, state.get('renderProperties').toJSON());
-  const data = transformedData.data;
-  const lookup = transformedData.lookup;
-  const newState = _.omit(action.newState, data);
```
Comment (andreme) [inline src/reducers/dataReducer.js]: _.omit() accepts an array and then tries to unset the each value in the array from the object

File: src/utils/dataUtils.js [hunk 2]
```diff
@@ -34,7 +34,10 @@ export function transformData(data, renderProperties) {
     throw new Error(`Griddle: Property '${renderProperties.rowProperties.rowKey}' doesn't exist in row data. Please specify a rowKey that exists in <RowDefinition>`);
   }
 
-  const transformedData = data.reduce(({ list, lookup }, rowData, index) => {
+  const list = [];
+  const lookup = {};
+
+  data.forEach((rowData, index) => {
```
Comment (andreme) [inline src/utils/dataUtils.js]: this reduces the time spent here for 400 records from 30ms to < 0.3ms

Comment (ryanlanciaux) [inline src/utils/dataUtils.js]: Awesome!

File: src/reducers/dataReducer.js (hunk 1 above)
Comment (dahlbyk) [inline src/reducers/dataReducer.js]: Great use of spread to refactor this!

**Comment by ryanlanciaux** [comment]
Thanks so much for adding this!

**Pull request merged**